homeassistant/helpers/selector.py

```diff
@@ -218,7 +219,12 @@ class AttributeSelector(Selector):
 
     selector_type = "attribute"
 
-    CONFIG_SCHEMA = vol.Schema({vol.Required("entity_id"): cv.entity_id})
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Required("entity_id"): cv.entity_id,
+            vol.Optional("exclude_attributes"): [str],
```

Comment:
If we don't want to validate in core:
```suggestion
            # exclude_attributes is used by frontend, not validated by core
            vol.Remove("exclude_attributes"): [str],
```

Reply:
It is not about being willing, we simply can't.

The attribute selector allows custom input, which includes ones that are not shown.